Lib/test/test_glob.py

```diff
@@ -511,6 +511,10 @@ def fn(pat):
 @skip_unless_symlink
 class SymlinkLoopGlobTests(unittest.TestCase):
 
+    # gh-109959: On Linux, glob._isdir() and glob._lexists() can return False
+    # randomly when checking the "link/" symbolic link.
+    # https://github.com/python/cpython/issues/109959#issuecomment-2577550700
+    @unittest.skipIf(True, "flaky test")
```

Comment:
```suggestion
    @unittest.skip("flaky test")
```

We can simplify this if you want: https://docs.python.org/3/library/unittest.html#unittest.skip.